### PR TITLE
Fix depends based build

### PR DESCRIPTION
**Pull request by fetzerch**

### CODE REVIEW

File: depends/common/moonlight-common-c/CMakeLists.txt [hunk 1]
```diff
@@ -23,23 +22,28 @@ set(MOONLIGHT_SOURCES
...
-
 include_directories(
-    ${OPENSSL_INCLUDE_DIRS}
+    ${OPENSSL_INCLUDE_DIR}
     enet/include
 )
 
-set(CMAKE_POSITION_INDEPENDENT_CODE ON) 
-add_library(${PROJECT_NAME} STATIC ${MOONLIGHT_SOURCES})
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```
Comment (garbear) [inline depends/common/moonlight-common-c/CMakeLists.txt:30]: Where is `ENET_SOURCES` defined?

File: Readme.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # game.moonlight
 
-A Moonlight client using the Kodi Game API
+A Moonlight client using the Kodi Game API.
+
+Moonlight is an open source implementation of NVIDIA GameStream and allows to
+stream your collection of Steam games from the gaming machine to the client.
```
Comment (garbear) [inline Readme.md]: leave the text wrapping to the markdown renderer

File: CMakeLists.txt [hunk 3]
```diff
@@ -1,56 +1,55 @@
...
+    ${MOONLIGHT_INCLUDE_DIRS}
+    ${PROJECT_SOURCE_DIR}/src
+)
 
 list(APPEND DEPLIBS
-            ${kodiplatform_LIBRARIES}
-            ${p8-platform_LIBRARIES}
-            ${OPENSSL_LIBRARIES}
-            ${CURL_LIBRARIES}
-            ${PUGIXML_LIBRARY}
```
Comment (garbear) [inline CMakeLists.txt:31]: Should this be `PUGIXML_LIBRARIES`?

**Comment by fetzerch** [comment]
Fixed, thanks. Convention is *_LIBRARIES. Unfortunately the built-in OpenSSL modules is one of the few that doesn't do it correctly.
Curl still needs to be moved to depends, but that will take a little longer, unless we have it already used in one of the addons.

**Pull request merged**